test/functorch/test_aotdispatch.py

```diff
@@ -756,8 +757,8 @@ def f(a):
         test = torch.ones(4, requires_grad=True) + 0
         test_view = test[0::2]
 
-        out_ref = f(ref_view)
-        out_test = f_compiled(test_view)
+        out_ref = f(ref_view)  # noqa: F841
+        out_test = f_compiled(test_view)  # noqa: F841
```

Comment:
@zou3519 I guess the line below is a typo and should check these two variables are equal?